src/app/components/header/Header.tsx

```diff
@@ -14,16 +15,35 @@ const styles = {
   }),
   icon: (theme: any) => ({
     color: `${theme.palette.WHITE}`,
-    paddingLeft: '20px',
   }),
   icons: {
     alignItems: 'center',
     display: 'flex',
     paddingRight: '20px',
   },
+  quantity: (theme: any) => ({
+    backgroundColor: `${theme.palette.DARK_BROWN}`,
+    display: 'flex',
+    borderRadius: '50%',
+    color: `${theme.palette.WHITE}`,
+    fontSize: '8px',
+    height: '16px',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '16px',
+    position: 'absolute',
+    top: '0',
+    right: '6px',
+  }),
 };
 
-const Header: React.FC = () => {
+const Header = () => {
+  const getCartQuantity = useCartStore((state) => {
```

Comment:
analogicznie jak w innym komentarzu raczej zrobiłbym na zewnątrz osobny hook `useCartQuantity`
 i tylko go to użył